Formula/pwntools.rb

```diff
@@ -160,28 +156,23 @@ class Pwntools < Formula
   end
 
   resource "ROPGadget" do
-    url "https://files.pythonhosted.org/packages/91/8d/4231736618976a335113efa3fccdf060765348b1d58e4bb50ccebf926800/ROPGadget-7.1.tar.gz"
-    sha256 "757f82670309898c8926194bef8912f7d22004fb8a7ef1897954610b80843d4f"
+    url "https://files.pythonhosted.org/packages/80/cc/476fd64384b2e238b984e13be09a068f2ff61917e53267c3274a3574c309/ROPGadget-7.2.tar.gz"
+    sha256 "ec705810d37043b2aca32a164416c6198de7770bb5e45ca1bc219acb87ee2073"
   end
 
   resource "rpyc" do
-    url "https://files.pythonhosted.org/packages/ec/94/1d46970dee93f13ee356f2a04242cae50cfdf1847cc30aaa575055f1679f/rpyc-5.2.3.tar.gz"
-    sha256 "220199bde0841374009b2693d8b9c7aa8bd170840813a22c594d30848ccbb27e"
+    url "https://files.pythonhosted.org/packages/36/56/3063981236a19c2a99a41f8184e7b34d4920668af74abe6b715e433f4a43/rpyc-5.3.0.tar.gz"
+    sha256 "cde4bd871c9008d16312ce7f432d4d50a38ddc300115045f1bbd78ba14651404"
   end
 
   resource "sortedcontainers" do
     url "https://files.pythonhosted.org/packages/e8/c4/ba2f8066cceb6f23394729afe52f3bf7adec04bf9ed2c820b39e19299111/sortedcontainers-2.4.0.tar.gz"
     sha256 "25caa5a06cc30b6b83d11423433f65d1f9d76c4c6a0c90e3379eaa43b9bfdb88"
   end
 
-  resource "unicorn" do
-    url "https://files.pythonhosted.org/packages/b3/9c/d6d3de658534bbc82fa73dea711860e345c5deec8974c5bd25588c399f9b/unicorn-2.0.1.tar.gz"
-    sha256 "62ad9bb3249ae2794b4cffe8d17dc0aef3f7db12a1938a7a63bccd4797bf4e30"
-  end
```

Comment:
Also still needed as `unicorn` formula is only the C library. Need this for python bindings.

Reply:
updated